tests/test_decoder.rs

```diff
@@ -72,6 +72,32 @@ fn test_basic_sourcemap_with_root() {
     assert!(iter.next().is_none());
 }
 
+#[test]
+fn test_basic_sourcemap_with_absolute_uri_root() {
+    let input: &[_] = b"{
```

Comment:
```suggestion
    let input: &[_] = rb#"{
```
so you won’t have to escape everything